File: inputs.py
#lee las dimensiones de la matriz desde la terminal
dimensions = input()
rows, cols = map(int, dimensions.split())

matrix = []

for i in range(rows):
    row = []
    for j in range(cols):
        element = int(input())
        row.append(element)
    matrix.append(row)

#print matrix
print("La matriz ingresada es:")
for row in matrix:
    print(row)

File: inputs.cpp
#include <iostream>
#include <vector>

int main() {
    int rows, cols;
    std::cin >> rows >> cols;

    std::vector<std::vector<int>> matrix(rows, std::vector<int>(cols));

    for (int i = 0; i < rows; i++){
        for (int j = 0; j < cols; j++){
            std::cin >> matrix[i][j];
        }
    }

    std::cout << "La matriz ingresada es: " << std::endl;
    for (int i = 0; i < rows; i++) {
        for (int j = 0; j < cols; j++){
            std::cout << matrix[i][j] << " ";
        }
        std::cout << std::endl;
    }

    return 0;
}






